### PR TITLE
cli: add the option to skip adding footer

### PR DESCRIPTION
**Pull request by jfilak**
I need merge only a class file with '@@abapmerge include' without
building a complete single source file program.

More precisely, I need to bundle the W3MI data in
the class ZCL_ABAPGIT_UI_FACTORY.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -78,7 +81,9 @@ class Logic {
     try {
       const parsedArgs = Logic.parseArgs();
       output = Merge.merge(Logic.readFiles(parsedArgs.entryDir), parsedArgs.entryObjectName, { skipFUGR: parsedArgs.skipFUGR });
-      output = Merge.appendFooter(output, PackageInfo.version);
+      if (!parsedArgs.noFooter) {
```
Comment (larshp) [inline src/cli.ts]: I'd be more comfortable with "parsedArgs.noFooter === true", as its not a double negation, plus handling of undefined in javascript is always tricky

Comment (larshp) [reply src/cli.ts]: actually `=== false`, well, yea, exactly

Comment (larshp) [reply src/cli.ts]: or well, I cant figure it out

Comment (jfilak) [reply src/cli.ts]: Good point. Amended.

**Comment by larshp** [comment]
0.12.1 published

**Comment by jfilak** [comment]
Thank you!

**Pull request merged**